src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -189,7 +189,7 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
         StringBuilder sb = new StringBuilder()
 
         if (resourceSet.isMemSet()) sb << " -l mem=" << resourceSet.getMem().toString(BufferUnit.M)
-        if (resourceSet.isWalltimeSet()) sb << " -l walltime=" << durationToPbsWallTime(resourceSet.getWalltime())
+        if (resourceSet.isWalltimeSet()) sb << " -l walltime=" << resourceSet.getWalltime()
```

Comment:
I think it is totally ok to have a conversionMethod in each JobManager, if it is necessary. So you could leave it here, but please unify it, if necessary. How is it with LSF? PBS and SGE share the same setup, how about SLURM?